src/connector/src/parser/unified/mod.rs

```diff
@@ -0,0 +1,35 @@
+use risingwave_common::types::{DataType, Datum};
+use thiserror::Error;
+
+pub mod avro;
+pub mod debezium;
+pub mod json;
+pub mod upsert;
+pub mod util;
+pub type AccessResult = std::result::Result<Datum, AccessError>;
+pub trait Access {
+    fn access(&self, path: &[&str], shape: Option<&DataType>) -> AccessResult;
+}
+#[derive(Debug)]
+pub enum RowOperation {
+    Insert,
+    Update,
```

Comment:
Consider renaming because `OperateRow` is confusing. How about `CdcEvent` or `ChangeEvent`, or anything else?